### PR TITLE
[Add a Play]: Schulte tables game - issue: #1381

### PR DESCRIPTION
**Pull request by mhShohan**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

[Add a Play]: Schulte tables game - issue: #1381
chulte tables are tables with randomly arranged characters, usually numbers used to check and develop the speed of their visual search in a specific order. These tables were originally developed by the German psychotherapist Walter Schulte as a psychodiagnostic test for studying the properties of attention.

Fixes # (issue) #1381

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [X ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/plays/schulte-tables/SchulteTables.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// WARNING: Do not change the entry componenet name
+function SchulteTables(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
```
Comment (priyankarpal) [inline src/plays/schulte-tables/SchulteTables.jsx:14]: Could you tell us about the comment?

File: src/plays/schulte-tables/cover.png
Comment (priyankarpal) [inline src/plays/schulte-tables/cover.png:1]: I will suggest to use CDN link for the image

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/plays/schulte-tables/SchulteTables.jsx (hunk 1 above)
Comment (mhShohan) [reply src/plays/schulte-tables/SchulteTables.jsx:14]: I didn't remove the initial comment mistakenly.

**Review by mhShohan — COMMENTED** [review]
removed comments in SchulteTable

File: src/plays/schulte-tables/utils/shuffleArray.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+const shuffle = () => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+
+  return array;
+};
+
+export default shuffle;
```
Comment (priyankarpal) [inline src/plays/schulte-tables/utils/shuffleArray.js:15]: you can shorten the code. `Array.from` to create an array of numbers from 1 to 25 and then shuffle it using the [Fisher-Yates algorithm](https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle)

```javascript
const shuffle = () => {
  const array = Array.from({ length: 25 }, (_, i) => i + 1);

  for (let i = array.length - 1; i > 0; i--) {
    const j = Math.floor(Math.random() * (i + 1));
    [array[i], array[j]] = [array[j], array[i]];
  }

  return array;
};

export default shuffle;
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by mhShohan — COMMENTED** [review]
shorten the code of shuffle function

**Review by priyankarpal — APPROVED** [review]

**Review by joshi-kaushal — APPROVED** [review]

**Pull request merged**